portfolio/src/main/webapp/blogs/blogs.css

```diff
@@ -1,45 +1,45 @@
 div {
-  width:100%;
+  width: 100%;
   text-align: center;
 }
- 
+
 h2 {
   font: 400 40px/1.5 Helvetica, Verdana, sans-serif;
   margin: 0;
   padding: 0;
 }
- 
+
 ul {
   list-style-type: none;
   margin: 0;
   padding: 0;
 }
- 
+
 li {
   font: 200 20px/1.5 Helvetica, Verdana, sans-serif;
   border-bottom: 1px solid #ccc;
-  width:100%;
+  width: 100%;
 }
- 
+
 li:last-child {
   border: none;
 }
- 
+
 li a {
   text-decoration: none;
```

Comment:
Nice job cleaning up your code — it looks neat! If you have time, it might be helpful to alphabetize the CSS properties in each block. :) Overall, great job!